Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN pip3 install -e python
 # Build Authenticator Go Binary
 WORKDIR /usr/src/approzium/authenticator
 COPY authenticator/ .
-RUN --mount=type=cache,target=$GOPATH/pkg/mod go build
+RUN mount=type=cache,target=$GOPATH/pkg/mod go build
```

Comment:
I am curious what this change does.

Reply:
I got an error that the "mount" parameter was unfound. By making it not a parameter, but a command, it began working for me. Not sure why it differs when running in our various environments.